viewer/packages/3d-overlays/src/OverlayPointsObject.ts

```diff
@@ -68,9 +69,25 @@ export class OverlayPointsObject extends Group {
       minPixelSize,
       maxPixelSize,
       radius,
-      colorTint
+      colorTint,
+      false
```

Comment:
Should `depthWrite` be `false` for both front and back? I thought it might be `true` for front?

Reply:
We decided to set it to false after our discussion, will be fixed later if we will want to address the performance problem and depth sorting bug.